notifications/constructor-insecure.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Notification constructor (basic)</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({ explicit_timeout: true })
+if (Notification.permission == "granted") {
+    alert("TEST NOT RUN. Change your browser settings so that notifications"
+        + " for this origin are cleared, and then reload this page.")
+} else {
+    async_test(function (t) {
+        t.step(function () {
+            var notification = new Notification("New Email Received")
+            notification.onerror = function(e) {
```

Comment:
You do need a t.step here. Probably t.step_func_done and then you can omit the t.done() as well.